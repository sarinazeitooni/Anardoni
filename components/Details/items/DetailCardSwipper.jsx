import ThirdSwipper from "../../thirdSwipper/ThirdSwipper";
import style from './scss/DetailCardSwipper.module.scss';
import messages from "./messages/messages";

const DetailCardSwipper = ({item})=>{
    return(
            <div className={style.preview}>
                <span className={style['preview-title']}>{messages.swipperTitle}</span>
                <ThirdSwipper item={item}/>
            </div>
    )
}
export default DetailCardSwipper;