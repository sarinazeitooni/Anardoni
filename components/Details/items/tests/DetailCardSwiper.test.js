import React from 'react';
import { shallow } from 'enzyme';
import DetailCardSwipper from "../DetailCardSwipper";
describe("DetailCardSwipper component" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<DetailCardSwipper/>)
        expect(container).toMatchSnapshot();
    })
});