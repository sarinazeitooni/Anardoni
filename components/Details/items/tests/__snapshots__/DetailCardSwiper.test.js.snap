// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailCardSwipper component it should render a div 1`] = `ShallowWrapper {}`;
